plugins/modules/notification/slack.py

```diff
@@ -31,7 +32,9 @@
         be ignored.  See token documentation for information.
   token:
     description:
-      - Slack integration token.  This authenticates you to the slack service.
+      - Slack integration token. This authenticates you to the slack service.
+        Make sure to use the correct type of token, depending on what method you use.
+      - Webhook token:
```

Comment:
```suggestion
      - "Webhook token:
```
Otherwise this isn't parsed correctly as YAML.